prime-router/settings/prod/0048-setup-clarkcounty-oh-doh.yml

```diff
@@ -0,0 +1,34 @@
+# ./prime multiple-settings set --env prod --input ./settings/prod/0048-setup-clarkcounty-oh.yml
+---
+- name: oh-clark-doh
+  description: Ohio Department of Health - Clark County
+  jurisdiction: COUNTY
+  stateCode: OH
+  countyName: Clark
+  receivers:
+    - name: elr-download
+      organizationName: oh-clark-doh
+      topic: covid-19
+      customerStatus: testing
+      jurisdictionalFilter:
+        - matches(ordering_facility_state, OH)
+        - matches(ordering_facility_county, Clark)
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        suppressQstForAoe: true
+        receivingApplicationName: OHDOH
+        receivingApplicationOID: 2.16.840.1.114222.4.1.3674
+        receivingFacilityName: OHDOH
+        receivingFacilityOID: 2.16.840.1.114222.4.1.3674
+        reportingFacilityName: CDC PRIME
+        reportingFacilityId: 36DSMP9999
+        nameFormat: ohio
+        suppressHl7Fields: OBX-23-11
+        # turn off UNK and ASKU for this field
+        useBlankInsteadOfUnknown: patient_race
+      timing:
+        operation: MERGE
+        numberPerDay: 1440 # Every minute
+        initialTime: 00:00
```

Comment:
Suggest that you want to create the at a different time than midnight eastern to spread out the load. 00:02 would be better.